### DISCUSSION TITLE
How do you get the count of the records in the title of a RelationManager?

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#6544] [Help]
Hey!

Little tricky question: how could I get the count of the records in the title of a RelationManager? I tried to call the owner record and count the relationship, but since the getTitle() function is static, I can't use that method.

Any idea?

Thanks!

**Comment by danharrin** [comment]
I think there is a `getTitleForRecord()` method we introduced that allows you to do this instead. They are being combined in v3

**Comment by NicolasSacC** [comment]
Thanks a lot!
________________________________
De : Dan Harrin ***@***.***>
Envoyé : Thursday, May 18, 2023 6:22:51 AM
À : filamentphp/filament ***@***.***>
Cc : NicolasSacC ***@***.***>; Author ***@***.***>
Objet : Re: [filamentphp/filament] How do you get the count of the records in the title of a RelationManager? (Discussion #6544)


I think there is a getTitleForRecord() method we introduced that allows you to do this instead. They are being combined in v3
